app/src/main/java/cm/aptoide/pt/promotions/ClaimStatusWrapper.java

```diff
@@ -0,0 +1,31 @@
+package cm.aptoide.pt.promotions;
+
+import java.util.List;
+
+public class ClaimStatusWrapper {
+
+  private final Status status;
+  private final List<Error> errors;
+
+  public ClaimStatusWrapper(Status status, List<Error> errors) {
+
+    this.status = status;
+    this.errors = errors;
+  }
+
+  public Status getStatus() {
+    return status;
+  }
+
+  public List<Error> getErrors() {
+    return errors;
+  }
+
+  public enum Status {
+    ok, fail
+  }
+
+  public enum Error {
+    promotionClaimed, wrongAddress, wrongCaptcha, generic
```

Comment:
Instances of an Enum are constants and should follow constants naming patterns, all uppercase.